kie-server-parent/kie-server-tests/kie-server-integ-tests-optaplanner/src/test/java/org/kie/server/integrationtests/optaplanner/OptaplannerIntegrationTest.java

```diff
@@ -275,6 +277,7 @@ public void testGetBestSolution() throws Exception {
         for(Object process : processList) {
             Object computer = valueOf(process, "computer");
             assertNotNull(computer);
+            assertTrue(computerList.contains(computer));
```

Comment:
This should actually be stronger than contains (which allows equals but not same objects too). The same instance should be in the computerList. But there's an open bug for JAXB (not for XStream) that that's not the case (see todo about @XmlID and @XmlIDREF).